packages/dioxus-blitz/src/window.rs

```diff
@@ -45,7 +97,20 @@ impl<'a, Doc: DocumentLike> View<'a, Doc> {
             None => false,
             Some(waker) => {
                 let cx = std::task::Context::from_waker(waker);
-                self.renderer.poll(cx)
+                if self.renderer.poll(cx) {
+                    #[cfg(feature = "accesskit")]
+                    {
+                        // TODO send fine grained accessibility tree updates.
+                        let changed = mem::take(&mut self.renderer.dom.as_mut().changed);
+                        if !changed.is_empty() {
+                            self.build_accessibility_tree()
+                        }
```

Comment:
Seems like it wouldn't be too hard to send a `TreeUpdate` with just changed nodes here?

Reply:
I think you're right this is how we should handle this, I was just having some trouble getting the hierarchy working. The issue I had was creating IDs for the accesskit elements so maybe re-using the Blitz element IDs here like you mentioned could help?